src/components/CartItem/CartItem.scss

```diff
@@ -80,11 +80,16 @@
     height: 32px;
     border: 1px solid $element-color;
     margin: 1.5%;
+    align-items: center; 
 }
 
 .cart-item__buttons-icon {
-    width: 16px;
-    height: 16px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border: none;
+    background: none;
+    outline: none;
```

Comment:
To center icon inside button simply add
```
display: flex;
align-items: center;
justify-content: center; 
```
to .cart-item__buttons-icons class

Reply:
![Monosnap Vite + React + TS - Google Chrome 2024-04](https://github.com/fs-jan24-codeup/phone-catalog/assets/155641185/210fb08a-0405-4967-b7c6-f14ee1feaf72)
It have default styles